### PR TITLE
Adds one-line description of images produced by this build

### PR DESCRIPTION
**Pull request by JBAhire**
- added service description to README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,3 +3,11 @@
 ###### org.hypertrace.viewgenerator
 
 [![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-view-generator.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-view-generator)
+
+This repo provides two jobs: *1) Hypertrace view creator job 2) Hypertrace view generator job*
+
+Hypertrace view creator:
+It is a bootstrap job that creates require views in pinot like spanEventView, backendEntityView, etc -
```
Comment (buchi-busireddy) [inline README.md:10]: s/require/required/

File: README.md [hunk 2]
```diff
@@ -3,3 +3,11 @@
 ###### org.hypertrace.viewgenerator
 
 [![CircleCI](https://circleci.com/gh/hypertrace/hypertrace-view-generator.svg?style=svg)](https://circleci.com/gh/hypertrace/hypertrace-view-generator)
+
+This repo provides two jobs: *1) Hypertrace view creator job 2) Hypertrace view generator job*
```
Comment (tim-mwangi) [inline README.md:7]: view-generator is really a background service

File: README.md (hunk 1 above)
Comment (adriancole) [reply README.md:10]: protip, unless reviewing on phone, where agree this is difficult, go ahead and click this to make feedback like this easy peasy.

Screen Shot 2020-08-19 at 9 20 35 AM

Ends up like this:

```suggestion
It is a bootstrap job that creates required views in pinot like spanEventView, backendEntityView, etc - 
```

Comment (buchi-busireddy) [reply README.md:10]: Ahh.. that's nice. Have never used this feature but it's nice.

**Pull request merged**